{
  "cveTags": [],
  "descriptions": [
    {
      "lang": "en",
      "value": "Asio C++ Library before 1.13.0 lacks a fallback error code in the case of SSL_ERROR_SYSCALL with no associated error information from the SSL library being used."
    },
    {
      "lang": "es",
      "value": "La librería Asio C++ anterior a 1.13.0 carece de un código de error de respaldo en el caso de SSL_ERROR_SYSCALL sin información de error asociada de la librería SSL que se está utilizando."
    }
  ],
  "id": "CVE-2019-25219",
  "lastModified": "2024-11-01T12:57:35.843",
  "metrics": {
    "cvssMetricV31": [
      {
        "cvssData": {
          "attackComplexity": "LOW",
          "attackVector": "NETWORK",
          "availabilityImpact": "NONE",
          "baseScore": 7.5,
          "baseSeverity": "HIGH",
          "confidentialityImpact": "HIGH",
          "integrityImpact": "NONE",
          "privilegesRequired": "NONE",
          "scope": "UNCHANGED",
          "userInteraction": "NONE",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
          "version": "3.1"
        },
        "exploitabilityScore": 3.9,
        "impactScore": 3.6,
        "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
        "type": "Secondary"
      }
    ]
  },
  "published": "2024-10-29T17:15:03.567",
  "references": [
    {
      "source": "cve@mitre.org",
      "url": "https://github.com/chriskohlhoff/asio/commit/93337cba7b013150f5aa6194393e1d94be2853ec"
    },
    {
      "source": "cve@mitre.org",
      "url": "https://github.com/chriskohlhoff/asio/compare/asio-1-12-2...asio-1-13-0"
    },
    {
      "source": "cve@mitre.org",
      "url": "https://think-async.com/Asio/"
    }
  ],
  "sourceIdentifier": "cve@mitre.org",
  "vulnStatus": "Awaiting Analysis",
  "weaknesses": [
    {
      "description": [
        {
          "lang": "en",
          "value": "CWE-1188"
        }
      ],
      "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
      "type": "Secondary"
    }
  ]
}